### ISSUE TITLE
Opa REPL should respect .inputrc semantics

### ISSUE THREAD

**Issue by danielpops** [open-policy-agent/opa#1503]
# Expected Behavior
If I have a .inputrc file active (e.g. `set editing-mode vi` to get vim bindings), the Opa REPL shell should respect bash's .inputrc settings, similar to how python, mysql, etc. other shells behave.

# Actual Behavior
vi bindings are not supported

# Steps to Reproduce the Problem

  1. Create a `.inputrc` in your home directory with the following contents:
`set editing-mode vi`
  2. `exec bash` (or equivalent) to activate. Observe vi mode ability to e.g. switch between normal vs insert mode is respected
  3. `opa run`. Observe vi mode ability to e.g. switch between normal vs insert mode is *not* respected

# Additional Info
this occurs in bash on ubuntu xenial and osx mojave

**Comment by tsandall** [comment]
@danielpops I agree, this would be nice, though so far, it hasn't been a priority. If someone wants to look into this, I'd be happy to answer questions and point in the right direction.

For context, OPA doesn't implement the interactive shell. It uses https://github.com/peterh/liner under the hood.

**Comment by danielpops** [comment]
Thanks @tsandall. I've submitted a similar bug report to the liner project.

**Comment by sspaink** [comment]
https://github.com/peterh/liner last merged PR was 5 years ago, maybe it is time to look into an alternative instead.

**Comment by sspaink** [comment]
with: https://github.com/open-policy-agent/opa/pull/8882 we now use [github.com/reeflective/readline ](https://github.com/reeflective/readline) 

and this new library supports .inputrc: https://github.com/reeflective/readline/wiki/Configuration-File#loading-configurations
